### PR TITLE
plans: show must set result type, default is varchar.

### PR DESCRIPTION
**Pull request by siddontang**
SQLAlchemy will use column type to do something, so we cannot return 0
for column type.

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: plan/plans/show_test.go [hunk 1]
```diff
@@ -76,6 +77,8 @@ func (p *testShowSuit) TestShowVariables(c *C) {
 	c.Assert(fls, HasLen, 2)
 	c.Assert(fls[0].Name, Equals, "Variable_name")
 	c.Assert(fls[1].Name, Equals, "Value")
+	c.Assert(fls[0].Col.Tp, Equals, mysql.TypeVarchar)
+	c.Assert(fls[0].Col.Tp, Equals, mysql.TypeVarchar)
```
Comment (qiuyesuifeng) [inline plan/plans/show_test.go]: This line can be removed?

Comment (qiuyesuifeng) [inline plan/plans/show_test.go]: Or c.Assert(fls[1].Col.Tp, Equals, mysql.TypeVarchar)?

**Comment by siddontang** [comment]
PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**